subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -11,6 +11,7 @@ config BT_LLL_VENDOR_NORDIC
 	select BT_HAS_HCI_VS
 	select ENTROPY_NRF5_RNG
 	select ENTROPY_NRF5_BIAS_CORRECTION
+	select BT_CTLR_CONN_RSSI if BT_CONN && BT_HCI_RAW
```

Comment:
Make it a new `BT_CTLR_CONN_RSSI_SUPPORT` Kconfig and be selected by `BT_CONN && BT_HCI_RAW` there. This way other vendor Kconfig get the support selected.

Reply:
Fixed